src/collectors/processcheck/processcheck.py

```diff
@@ -0,0 +1,183 @@
+# coding=utf-8
+
+"""
+A Diamond collector that checks process defined in it's
+config file by matching them with their executable filepath or the process name.
+
+Example config file ProcessCheckCollector.conf
+
+```
+enabled=True
+ttl=150
+[process]
+[[postgres]]
+exe=^\/usr\/lib\/postgresql\/+d.+d\/bin\/postgres$
+name=^postgres,^pg
+
+```
+exe and name are both lists of comma-separated regexps.
+
+"""
+
+import re
+
+import diamond.collector
+import diamond.convertor
+
+try:
+    import psutil
+    psutil
```

Comment:
What does this additional call after the import do? Initialize it?

Reply:
good question, not sure why it is there, I'll remove.